### ISSUE TITLE
​Blank screen on mobile when sharing

### ISSUE THREAD

**Issue by artwist-polyakov** [toeverything/AFFiNE#8338]
### What happened?

On mobile browsers, when attempting to open shared notes, a blank screen is displayed.

**Steps to Reproduce:**
1. Open Chrome browser on a mobile device.
2. Go to [this link](https://docs.polyakov.marketing/workspace/8f799969-8391-40eb-a85e-722cd6146c06/2wXXc3IsiRkYXavczZcUA?mode=page).  https://docs.polyakov.marketing/workspace/8f799969-8391-40eb-a85e-722cd6146c06/2wXXc3IsiRkYXavczZcUA?mode=page
3. Observe the blank screen.

**Additional Information:**
- The issue only occurs in notes created in AFFiNE version 0.17. Previous stable versions work correctly.

logs in console:
![CleanShot 2024-09-20 at 18 26 57@2x](https://github.com/user-attachments/assets/ffa22a5b-c46f-492c-83d5-a5dc643101cb)

large screenshot from iPhone
![AFFiNE](https://github.com/user-attachments/assets/ffce3031-14eb-4549-a383-347b6b547b6a)

on desktops:
![CleanShot 2024-09-20 at 18 17 53@2x](https://github.com/user-attachments/assets/5d01f7d9-0ea3-44f3-ba9a-e9cb7504e571)

### Distribution version

Web (app.affine.pro)

### What browsers are you seeing the problem on if you're using web version?

Chrome

### Are you self-hosting?

- [X] Yes

**Comment by artwist-polyakov** [comment]
instead of the JS and CSS code, an error page is returned:
https://docs.polyakov.marketing/js/runtime.3b180e24.js
https://docs.polyakov.marketing/vendor.4b03f911.css
and this is the case for all mobile version resources.
For comparison, the desktop script: https://docs.polyakov.marketing/js/runtime.a8387fa9.js

Most likely, the problem is with the engine, since the URLs are fundamentally the same, i.e., nginx serves them from the same place, and the issue is that the mobile ones are somehow not there.
Or it could be something related to access rights.

**Comment by PengMai1998** [comment]
> instead of the JS and CSS code, an error page is returned: https://docs.polyakov.marketing/js/runtime.3b180e24.js https://docs.polyakov.marketing/vendor.4b03f911.css and this is the case for all mobile version resources. For comparison, the desktop script: https://docs.polyakov.marketing/js/runtime.a8387fa9.js
> 
> Most likely, the problem is with the engine, since the URLs are fundamentally the same, i.e., nginx serves them from the same place, and the issue is that the mobile ones are somehow not there. Or it could be something related to access rights.

Hello, how is this problem solved?

**Comment by artwist-polyakov** [comment]
> > instead of the JS and CSS code, an error page is returned: https://docs.polyakov.marketing/js/runtime.3b180e24.js https://docs.polyakov.marketing/vendor.4b03f911.css and this is the case for all mobile version resources. For comparison, the desktop script: https://docs.polyakov.marketing/js/runtime.a8387fa9.js
> > Most likely, the problem is with the engine, since the URLs are fundamentally the same, i.e., nginx serves them from the same place, and the issue is that the mobile ones are somehow not there. Or it could be something related to access rights.
> 
> Hello, how is this problem solved?

It isn't solved.

**Comment by artwist-polyakov** [comment]
@forehalo hello, problem still be represented after update to last stable. Why do you close this issue?

**Comment by fourdim** [comment]
> @forehalo hello, problem still be represented after update to last stable. Why do you close this issue?

It's fixed but not yet released in stable. Once linked PR is merged to the main branch, GitHub will automatically close the issue for you.

![image](https://github.com/user-attachments/assets/79339c25-e0ad-47b5-ac08-169a004f5c41)

That's how GitHub works.

For the stable releases:
For a version X.Y.Z:
When X, Y increases, the release will include all the main branch commits in the normal case.
When Z increases, only those **urgent** bug fixes will be included.

**Comment by artwist-polyakov** [comment]
@fourdim Got it! Still waiting next stable)

**Comment by artwist-polyakov** [comment]
@fourdim hello, i've just update version to last stable, and bug with mobile version of pages still being represented

**Comment by fourdim** [comment]
> @fourdim hello, i've just update version to last stable, and bug with mobile version of pages still being represented

0.17.5 is still a patch release
0.18.0

**Comment by ATPs** [comment]
I tried to use the beta version of 0.18.0. I can view the link, but it requires to login. Use firefox will be OK for mobile view.

**Comment by artwist-polyakov** [comment]
> > @fourdim hello, i've just update version to last stable, and bug with mobile version of pages still being represented
> 
> 0.17.5 is still a patch release 0.18.0

Hello, I’ve just launched last stable (today) and bug still being represented

**Comment by fourdim** [comment]
> > > @fourdim hello, i've just update version to last stable, and bug with mobile version of pages still being represented
> > 
> > 
> > 0.17.5 is still a patch release 0.18.0
> 
> Hello, I’ve just launched last stable (today) and bug still being represented

We didn't release 0.18.0 stable as of today, how?

**Comment by artwist-polyakov** [comment]
> > > > @fourdim hello, i've just update version to last stable, and bug with mobile version of pages still being represented
> > > 
> > > 
> > > 0.17.5 is still a patch release 0.18.0
> > 
> > 
> > Hello, I’ve just launched last stable (today) and bug still being represented
> 
> We didn't release 0.18.0 stable as of today, how?

Dont know, but errors still there
![CleanShot 2024-11-12 at 14 56 15@2x](https://github.com/user-attachments/assets/c91f42a2-fea7-49c1-ba0d-cbd1533fa0d6)

**Comment by artwist-polyakov** [comment]
@fourdim  Now it is fixed 
thank you for yours help

**Comment by ATPs** [comment]
I just updated to 0.18.0, self-host. the shared page, require to login
![image](https://github.com/user-attachments/assets/1c722c3a-9130-48d1-b6ae-2b1103feb857)

**Comment by artwist-polyakov** [comment]
> I just updated to 0.18.0, self-host. the shared page, require to login !

hello, I think you have to create new issue for this problem

current issue is solved and doesn’t correspond with bug you describe

**Comment by ATPs** [comment]
sure. thank you
